tree-diff

```diff
@@ -39,7 +39,7 @@ def selinux_diff(path1, path2, path, differences):
         return True
     if label1 != label2:
         props = differences.setdefault(path, {})
-        props["selinux"] = [os.fsdecode(label1), os.fsdecode(label2)]
+        props["selinux"] = [label1.strip('\n\0'), label2.strip('\n\0')]
```

Comment:
Hm, maybe we should just treat these as byte strings?

Reply:
What do you mean by byte strings?

Reply:
Ugh, I misread the code. Ignore me :)